tools/deprecate_version/deprecate_version.py

```diff
@@ -135,9 +137,16 @@ def GetRuntimeAlreadyTrue():
   with open('source/common/runtime/runtime_features.cc', 'r') as features:
     for line in features.readlines():
       match = runtime_features.match(line)
-      if match and 'test_feature_true' not in match.group(1):
-        print("Found existing flag " + match.group(1))
-        runtime_already_true.append(match.group(1))
+      # hit the sentinel value: no more true flags.
```

Comment:
given that we use the test_feature_flag as a guard and it depends on the ordering of the cc file, can we put a warning in that file mentioning this? Or should we separate the cc file into several files so each of them only has the features we want? Don't know if that is too heavy handed and I am being paranoid.